docs/src/docs/asciidoc/user/xml.adoc

```diff
@@ -35,6 +35,21 @@ These `Service` instances can then be used by `Cache` instances managed by the `
 extension point of the XML configuration (and Ehcache 3's modular architecture), as explained in the
 link:107{outfilesuffix}#_supplement_jsr_107_s_configurations[JSR-107 configuration section].
 
+=== `<default-serializers>` element
+
+A `<default-serializers>` element represents `Serializers` configured at `CacheManager` level. 
```

Comment:
Update this to reflect it contains a collection of elements doing what you explain